pkg/scheduler/metrics/metrics.go

```diff
@@ -97,13 +97,20 @@ var (
 			StabilityLevel: metrics.STABLE,
 		}, []string{"queue"})
 	SchedulerGoroutines = metrics.NewGaugeVec(
```

Comment:
Add a comment explaining that SchedulerGoroutines might not contain all the routines